lib/plugins/aws/customResources/generateZip.js

```diff
@@ -22,9 +21,6 @@ const ensureCustomResourcesArtifact = getEnsureArtifact(artifactName, cachePath
     return fse
       .copy(srcDirPath, tmpInstalledLambdaPath)
       .then(() => npmCommandDeferred)
```

Comment:
This line also should be removed

Reply:
Which line exactly? It's not highlighted.